### PR TITLE
added "operator_service_url" to operator claim

### PR DESCRIPTION
**Pull request by aricart**
operator_service_url enables an operator jwt to hint at service url where a NATS client can initiate a connection

### CODE REVIEW

**Comment by aricart** [comment]
The account server url is http and for interacting with an account server, operator service url is for tooling (nats-pub) in nsc to be able to tell where to connect.

File: operator_claims.go [hunk 1]
```diff
@@ -19,21 +19,26 @@ import (
...
 
 // Operator specific claims
 type Operator struct {
-	Identities       []Identity `json:"identity,omitempty"`
-	SigningKeys      StringList `json:"signing_keys,omitempty"`
-	AccountServerURL string     `json:"account_server_url,omitempty"`
+	Identities         []Identity `json:"identity,omitempty"`
+	SigningKeys        StringList `json:"signing_keys,omitempty"`
+	AccountServerURL   string     `json:"account_server_url,omitempty"`
+	OperatorServiceURL string     `json:"operator_service_url,omitempty"`
```
Comment (derekcollison) [inline operator_claims.go]: If this is where you can connect, should be an array. Even with NGS we could list main URL but then region and CP specific ones as well.

Comment (aricart) [reply operator_claims.go]: Fixed!

File: operator_claims.go [hunk 2]
```diff
@@ -19,24 +19,31 @@ import (
...
 	"net/url"
+	"strings"
 
 	"github.com/nats-io/nkeys"
 )
 
 // Operator specific claims
 type Operator struct {
-	Identities       []Identity `json:"identity,omitempty"`
-	SigningKeys      StringList `json:"signing_keys,omitempty"`
```
Comment (derekcollison) [inline operator_claims.go]: I prefer not typing things that are basic Go, should just be []string.

Comment (aricart) [reply operator_claims.go]: we'll have to add the uniquing code, empty string suppression, everywhere or at least on generate. Also all the clients will have to handle the removal of an element from the slice etc.

Comment (derekcollison) [reply operator_claims.go]: ok

**Pull request merged**